### ISSUE TITLE
`doorstop export all` doesn't do the obvious thing

### ISSUE THREAD

**Issue by robtaylor** [doorstop-dev/doorstop#678]
To export all, you still need to pass the path, e.g.
`doorstop export -x all .`
whereas, e.g.
`doorstop export -x all`
gives the error `doorstop: error: only single documents can be displayed`, which seems nonsensical from a user perspective.

I suggest `doorstop export all` should do the obvious thing of exporting all to the current working directory. @jacebrowning what do you think?

**Comment by jacebrowning** [comment]
I believe the `.` in `doorstop export all .` is the location we want the full document tree to be exported to. Is that not the case?

**Comment by robtaylor** [comment]
Yep. They issue here is the error or behaviour when that isn’t there.

**Comment by robtaylor** [comment]
Simplest fix is to fix error to ‘location for output file(s) is required’
though maybe a more ‘normal’ command line behaviour would be to output to
cwd and have a -o option for an output location, but this wouldn’t be
backward compatible.

**Comment by robtaylor** [comment]
Updated the description, will file a PR asap..
